src/Tribe/Service_Providers/Context.php

```diff
@@ -294,6 +317,11 @@ static function ( $post_type ) {
 				],
 				'organizer_post_type'  => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								return ! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) ) ?: Tribe__Context::NOT_FOUND;
```

Comment:
:no_entry_sign: Using short ternaries is not allowed

<sub>hash: f677f654ca87bf50b9933d325bacdd9f</sub>